File: VSExplorer/bin/Debug/s/Templates-Data/ItemTemplates/Web/CSharp/1033/SignalRPersistentConnectionItemTemplateWebv5.0.cs/MyConnection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Web;
using Microsoft.AspNet.SignalR;

public class $safeitemname$ : PersistentConnection
{
    protected override Task OnConnected(IRequest request, string connectionId)
    {
        return Connection.Send(connectionId, "Welcome!");
    }

    protected override Task OnReceived(IRequest request, string connectionId, string data)
    {
        return Connection.Broadcast(data);
    }
}

File: VSExplorer/bin/Debug/s/Templates-Data/ItemTemplates/Web/VisualBasic/1033/SignalRPersistentConnectionItemTemplateWebv5.0.vb/MyConnection.vb
Imports System.Threading.Tasks
Imports Microsoft.AspNet.SignalR

Public Class $safeitemname$
    Inherits PersistentConnection

    Protected Overrides Function OnConnected(ByVal request As IRequest, ByVal connectionId As String) As Task
        Return Connection.Send(connectionId, "Welcome!")
    End Function

    Protected Overrides Function OnReceived(ByVal request As IRequest, ByVal connectionId As String, ByVal data As String) As Task
        Return Connection.Broadcast(data)
    End Function
End Class
